hack/non-olm-install/README.md

```diff
@@ -0,0 +1,85 @@
+### Non OLM based operator installation
+
+The purpose of this script is to install and uninstall the GitOps Operator without using OLM. It uses latest version of the kustomize manifests available in the repository.
+
+### Prerequisites
+- kustomize (v4.57 or later)
+- kubectl (v1.26.0 or later)
+- yq (v4.31.2 or later)
+- bash (v5.0 or later)
+
+### Environment Variables
+The following environment variables can be set to configure various options for the installation/uninstallation process.
+
+#### Variables for Operator image and related manifests
+|Environment|Description|Default Value|
+|:----------|:---------:|:-----------:|
+|**NAMESPACE_PREFIX**|Namespace prefix to be used in the kustomization.yaml file when running kustomize|gitops-operator-|
+|**GIT_REVISION**|The revision of the kustomize manifest to be used.|master|
+|**OPERATOR_REGISTRY**|Registry server for downloading the container images|registry.redhat.io|
+|**GITOPS_OPERATOR_VER**|Version of the gitops operator version to use|1.8.1-1|
```

Comment:
The `latest` tag did not work. I wanted to use a default value that works for all. So wanted it to be a publicly available image. Since nightly build images are not available in `registry.redhat.io`, I am using `1.8.1`. This has disadvantage of getting outdated soon.